docs/ref/csrf.txt

```diff
@@ -257,9 +257,12 @@ The CSRF protection is based on the following things:
    due to the fact that HTTP 'Set-Cookie' headers are (unfortunately) accepted
    by clients that are talking to a site under HTTPS.  (Referer checking is not
    done for HTTP requests because the presence of the Referer header is not
-   reliable enough under HTTP.)
+   reliable enough under HTTP.)  The referer is compared against the 
+   ``conf.settings.CSRF_COOKIE_DOMAIN`` setting.  The ``CSRF_COOKIE_DOMAIN`` 
+   setting supports subdomains.  For example, ".example.com" will allow 
+   post requests from "www.example.com" and "api.example.com."
```

Comment:
This paragraph should also explain the behavior with the `CSRF_COOKIE_DOMAIN` default value of `None` (that is, that the Referer must match the `Host` header). Proposed sentence to tack on the end:

"If `CSRF_COOKIE_DOMAIN` is not set, the referer must match the `Host` header."